data.sql

```diff
@@ -7,3 +7,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 (4, 'Devimon', '2017-05-12', 5, true, 11.00)
 ;
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES 
+(5, 'Charmander', '2020-02-08', 0, false, -11),
+(6, 'Plantmon', '2021-11-15', 2, true, -5.7),
```

Comment:
- [ ] date_of_birth for `Plantmon` is 2022/11/15.

Reply:
My dear friend... My eyes are not good enough, so I cannot see the **2022**  :smile: 
![image](https://user-images.githubusercontent.com/100373071/177407898-420aa43d-7372-477a-a810-f91c072d91e3.png)

Reply:
The previous reviewer (@arslanbisharat) is asking to change the date of birth for **Plantmon** to **2022**, but according to the requirement's document, it should be **2021**. Following is the screenshot from the requirement's document
![image](https://user-images.githubusercontent.com/100373071/177410136-1aaefde2-f76c-41de-9bf9-68fb5a8cd3b4.png)

Reply:
I saw this issue after your project review. There are some confusions between our guidelines and yours so I have reported that to team